### PR TITLE
Update missing Chinese translations of v2.8 and latest versions

### PR DESCRIPTION
**Pull request by rootwuj**
## Description

Update Chinese translations of the `latest` and `v2.8` versions.

Modified files of version `latest` & `v2.8`:

- reference-guides/user-settings/user-settings.md
- reference-guides/cli-with-rancher/cli-with-rancher.md
- reference-guides/about-the-api/about-the-api.md
- reference-guides/rancher-security/rancher-security.md
- reference-guides/rancher-security/hardening-guides/hardening-guides.md
- reference-guides/rancher-security/hardening-guides/rke1-hardening-guide/rke1-hardening-guide.md
- reference-guides/rancher-security/hardening-guides/rke1-hardening-guide/rke1-self-assessment-guide-with-cis-v1.23-k8s-v1.23.md
- reference-guides/rancher-security/hardening-guides/rke1-hardening-guide/rke1-self-assessment-guide-with-cis-v1.24-k8s-v1.24.md
- reference-guides/rancher-security/hardening-guides/rke1-hardening-guide/rke1-self-assessment-guide-with-cis-v1.7-k8s-v1.25-v1.26-v1.27.md
- reference-guides/rancher-security/hardening-guides/rke2-hardening-guide/rke2-hardening-guide.md
- reference-guides/rancher-security/hardening-guides/rke2-hardening-guide/rke2-self-assessment-guide-with-cis-v1.23-k8s-v1.23.md
- reference-guides/rancher-security/hardening-guides/rke2-hardening-guide/rke2-self-assessment-guide-with-cis-v1.24-k8s-v1.24.md
- reference-guides/rancher-security/hardening-guides/rke2-hardening-guide/rke2-self-assessment-guide-with-cis-v1.7-k8s-v1.25-v1.26-v1.27.md
- reference-guides/rancher-security/hardening-guides/k3s-hardening-guide/k3s-hardening-guide.md
- reference-guides/rancher-security/hardening-guides/k3s-hardening-guide/k3s-self-assessment-guide-with-cis-v1.23-k8s-v1.23.md
- reference-guides/rancher-security/hardening-guides/k3s-hardening-guide/k3s-self-assessment-guide-with-cis-v1.24-k8s-v1.24.md
- reference-guides/rancher-security/hardening-guides/k3s-hardening-guide/k3s-self-assessment-guide-with-cis-v1.7-k8s-v1.25-v1.26-v1.27.md
- reference-guides/rancher-security/selinux-rpm/selinux-rpm.md
- reference-guides/rancher-security/rancher-security-best-practices.md
- reference-guides/rancher-security/security-advisories-and-cves.md
- reference-guides/rancher-security/rancher-webhook-hardening.md

### CODE REVIEW

**Comment by rootwuj** [comment]
@JacieChao Hi, could you please help me to review this PR? Thanks!

File: i18n/zh/docusaurus-plugin-content-docs/current/reference-guides/rancher-security/rancher-security.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+<p style={{padding: '8px'}}>请将安全问题发送至 <a href="mailto:security-rancher@suse.com">security-rancher@suse.com</a>。</p>
+</td>
+<td width="30%" style={{border: 'none'}}>
+<h4>公告</h4>
+<p style={{padding:'8px'}}>订阅 <a href="https://forums.rancher.com/c/announcements">Rancher 公告论坛</a>以获取版本更新。</p>
+</td>
+</tr>
+</table>
+
+安全是 Rancher 全部功能的基础。Rancher 集成了全部主流身份验证工具和服务，并提供了企业级的 [RBAC 功能](../../how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/manage-role-based-access-control-rbac/manage-role-based-access-control-rbac.md)，让你的 Kubernetes 集群更加安全。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference-guides/rancher-security/rancher-security.md:26]: 主流身份验证工具 => 主流认证工具

File: i18n/zh/docusaurus-plugin-content-docs/current/reference-guides/rancher-security/rancher-security.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+[安全增强型 Linux (SELinux)](https://en.wikipedia.org/wiki/Security-Enhanced_Linux) 是对 Linux 的安全增强。被政府机构使用之后，SELinux 已成为行业标准，并在 CentOS 7 和 8 上默认启用。
+
+我们提供了 `rancher-selinux` 和 `rke2-selinux` 两个 RPM（Red Hat 软件包），让 Rancher 产品能够在 SELinux 主机上正常运行。有关详细信息，请参阅[此页面](selinux-rpm/selinux-rpm.md)。
+
+### Rancher 加固指南
+
+Rancher 加固指南基于 <a href="https://www.cisecurity.org/benchmark/kubernetes/" target="_blank">CIS Kubernetes Benchmark</a>。
+
+加固指南为加固 Rancher 的生产安装提供了说明性指导。有关安全管控的完整列表，请参阅 Rancher 的 [CIS Kubernetes Benchmark自我评估](#cis-benchmark-和自我评估)指南。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference-guides/rancher-security/rancher-security.md:60]: Benchmark 格式空格

File: i18n/zh/docusaurus-plugin-content-docs/current/reference-guides/rancher-security/rancher-security.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+请注意，新报告不再共享或公开发布。
+
+### Rancher 安全公告和 CVE
+
+Rancher 致力于向社区通报我们产品中的安全问题。有关我们已解决的问题的 CVE（常见漏洞和暴露）列表，请参阅[此页](security-advisories-and-cves.md)。
+
+### Kubernetes 安全最佳实践
+
+有关保护 Kubernetes 集群的建议，请参阅 [Kubernetes 安全最佳实践](kubernetes-security-best-practices.md) 指南。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference-guides/rancher-security/rancher-security.md:91]: 指南前不需要空格，下同

**Review by JacieChao — COMMENTED** [review]
Please update both latest and v2.8 docs

**Review by JacieChao — COMMENTED** [review]
LGTM.
PR will be ready after other dependencies are ready

**Review by martyav — APPROVED** [review]

**Pull request merged**